packages/tools/client-debugger/client-debugger-view/src/components/WaitingLabels.tsx

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Type of {@link waitingLabels}.
+ */
+export type WaitingLabelsType = Record<string, string>;
```

Comment:
I don't think this is needed. Anyone referring to the `waitingLabels` const will get the inline typing information without this.